src/main/java/com/monumental/services/MonumentService.java

```diff
@@ -684,10 +684,12 @@ private Predicate buildDecadeQuery(CriteriaBuilder builder, Root root, Integer d
 
     /**
      * Gathers the various statistics related to Monuments for the About Page
+     * @param searchForSpecificMonuments - True to also include searching for the specific Monuments we display links
+     * to on the About Page, such as the 9/11 Memorial, False otherwise
```

Comment:
You might want to add that this override mostly exists because of an H2 limitation